src/client/datascience/notebook/fontAwesomeLoader.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const getPublicPath = () => {
+    return new URL(scriptUrl.replace(/[^/]+$/, '')).toString();
+};
+const uri = getPublicPath() + '../notebook/node_modules/font-awesome/css/font-awesome.min.css';
```

Comment:
Anytime I see paths, just make sure that it works on Windows + Mac / Linux.

Reply:
Curious why do we need `../notebook/node_modules`, isn't `node_modules/font...` sufficient?